action/protocol/poll/protocol.go

```diff
@@ -123,48 +132,62 @@ func NewProtocol(
 	if cfg.Consensus.Scheme != config.RollDPoSScheme {
 		return nil, nil
 	}
-	if !genesisConfig.EnableGravityChainVoting || electionCommittee == nil {
+
+	switch genesisConfig.PollMode {
+	case _modeLifeLong:
 		delegates := genesisConfig.Delegates
 		if uint64(len(delegates)) < genesisConfig.NumDelegates {
 			return nil, errors.New("invalid delegate address in genesis block")
 		}
 		return NewLifeLongDelegatesProtocol(delegates), nil
+	case _modeGovernance:
```

Comment:
fallthrough means governance only won't be used
we can remove this mode?

Reply:
currently it is controlled within `staking committee`  if we don't enable native staking contract, it is governance only mode

Reply:
i mean, both the "_modeGovernance" and "mixed" mode create the exactly same staking committee struct, so those 2 modes are actually identical

Reply:
i think eventhrough the code here is the same, we still have two different mode. They are not the same mode on our chain, not sure it is correct to combine them just because the code here is the same.